core-domain/src/main/java/org/uniprot/core/uniparc/UniParcEntryLight.java

```diff
@@ -0,0 +1,36 @@
+package org.uniprot.core.uniparc;
+
+import org.uniprot.core.Sequence;
+import org.uniprot.core.util.Pair;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.List;
+
+public interface UniParcEntryLight extends Serializable {
+    UniParcId getUniParcId();
+
+    List<String> getUniParcCrossReferences();
+    List<Pair<Integer, String>> getCommonTaxons();// taxonId and name pair
+
+    List<String> getUniProtKBAccessions();
+
+    String getUniProtExclusionReason();
+
+    Sequence getSequence();
+
+    List<SequenceFeature> getSequenceFeatures();
+
+    LocalDate getOldestCrossRefCreated();
+
+    LocalDate getMostRecentCrossRefUpdated();
+
+    // lazily loaded fields
+    List<Pair<Integer, String>> getOrganisms();// organism id and name pair
+
+    List<String> getProteinNames();
+
+    List<String> getGeneNames();
+
+    List<String> getProteomeIds();
```

Comment:
I think proteomes will need to be a pair as well, id and component, and I think that sometimes we do not have the component.  (https://www.uniprot.org/uniparc?fields=upi%2Corganism%2Clength%2Caccession%2Cfirst_seen%2Clast_seen%2Cproteome&query=P05067&view=table)

Reply:
Do you think we should rename it just to getProteomes() ?

Reply:
Based on our discussion, we agree to return proteome id for now. If asked, we will add it.